<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>775</width>
    <height>578</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Patient</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icons/edit.png</normaloff>icons/edit.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QPushButton" name="pushButton_2">
     <property name="font">
      <font>
       <pointsize>21</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Edit Patient</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>icons/edit.png</normaloff>icons/edit.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>751</width>
        <height>445</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string> ID :</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="id">
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="font">
              <font>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="text">
              <string> Name :</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="name">
             <property name="font">
              <font>
               <pointsize>11</pointsize>
              </font>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string> Age :</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="age">
             <property name="minimumSize">
              <size>
               <width>30</width>
               <height>1</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Gender :</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="gender">
             <property name="minimumSize">
              <size>
               <width>150</width>
               <height>3</height>
              </size>
             </property>
             <item>
              <property name="text">
               <string>Male</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Female</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Mobile :</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="mobile">
             <property name="minimumSize">
              <size>
               <width>300</width>
               <height>1</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>1</height>
              </size>
             </property>
             <property name="text">
              <string> Address :</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="address"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string> Status :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radio_new">
           <property name="text">
            <string> New</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radio_exiting">
           <property name="text">
            <string>Existing</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string> Comming From :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="comming_from"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string> Height :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="height"/>
         </item>
         <item>
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string> Weight :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="weight"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string> Disease :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="disease"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string> Diagnosis :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="diagnosis"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QPushButton" name="editpatient">
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>1000</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(78, 154, 6);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Save Changes</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
